metrics/report/report_dockerfile/fio-reads.R

```diff
@@ -224,7 +224,11 @@ read_clat_line_plot <- ggplot() +
 		legend.title=element_text(size=5),
 		legend.text=element_text(size=5),
 		legend.background = element_rect(fill=alpha('blue', 0.2))
-	)
+	) +
+	# We know this is a 0-100% x-axis, so make nice divisions, as the interesting results
+	# tend to be scrunched up at the 95+% end, and it's just easier to eyball them than with
+	# the default (divide into eight) divisions.
```

Comment:
Should this be divide into ten?

Reply:
the 'divide into eight' is describing what the default did - not what the new addition does.
The new addition does generate 10 tickmarks, but the graph rendering also renders sub-divisions, so we actually end up with 20 divisions (a line at every 5% mark).
I can try to rewrite and clarify that comment if you want?